lldb/test/Shell/SymbolFile/PDB/add-symbols.cpp

```diff
@@ -0,0 +1,39 @@
+// REQUIRES: lld, target-windows
```

Comment:
Does it need lld?

Doesn't really matter though, it wants clang-cl and lld will be next to it anyway.

Reply:
On Windows, it wouldn't require LLD, but on other platforms it requires `lld-link` to be available ([used as the linker](https://github.com/llvm/llvm-project/blob/346f48ecbcd5a2ba63b3947f3593acce2867692b/lldb/test/Shell/helper/build.py#L378-L382)), because we want to build a PE/COFF binary. So technically it requires `target-windows, (lld || system-windows)`. I think our tests already require LLD on Windows, so `system-windows` implies `lld`.